### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Marcosnr97**
## Summary
- Added html and css style for navbar in books list

## Screenshot
image

## Trello Card
https://trello.com/c/oLQenb8e/31-navbar

### CODE REVIEW

File: training-maquetado/scss/list.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 .book-list {
+  background-color: $wildsand;
   display: flex;
   flex-wrap: wrap;
-  max-width: 1366px;
-  width: 100%;
+  padding: $navbar-height + 36px 234px;
```
Comment (Marcosnr97) [inline training-maquetado/scss/list.scss]: I'm not sure why is this working or if this is correct

File: training-maquetado/book-list.html [hunk 2]
```diff
@@ -11,6 +11,10 @@
 </head>
 
 <body>
+    <div class="navbar">
+        <img src="assets/logo-wolox.png" class="navbar-logo" alt="Wolox logo" />
+        <button id="logout" class="navbar-logout-button text-small">Logout</button>
```
Comment (meelrossi) [inline training-maquetado/book-list.html]: why do you need the id here? also set the button type: ```<button type='button' ....```

File: training-maquetado/scss/list.scss (hunk 1 above)
Comment (meelrossi) [reply training-maquetado/scss/list.scss]: mmm this is weird. why do you need this?

File: training-maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@import './variables/sizes';  
+
+.navbar-logo {
+    align-self: center;
```
Comment (meelrossi) [inline training-maquetado/scss/navbar.scss]: check identation

File: training-maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    height: $navbar-height;
+  }
+  
+  .navbar-logout-button {
+    align-self: center;
```
Comment (meelrossi) [inline training-maquetado/scss/navbar.scss]: align them with the continaer. you shouldn't need the self-align

**Review by meelrossi — CHANGES_REQUESTED** [review]

**Review by meelrossi — APPROVED** [review]

**Pull request merged**